services/src/handlers/wms.rs

```diff
@@ -454,26 +447,11 @@ mod tests {
 
     #[test]
     fn test_wms_url() {
-        fn assert_wms_base(base: &str, result: &str) {
-            let base = Url::parse(base).unwrap();
-            let result = Url::parse(result).unwrap();
+        let workflow_id = WorkflowId::from_str("df756642-c5a3-4d72-8ad7-629d312ae993").unwrap();
 
-            let workflow_id = WorkflowId::from_str("df756642-c5a3-4d72-8ad7-629d312ae993").unwrap();
-
-            assert_eq!(wms_url(&base, workflow_id).unwrap(), result);
-        }
-
-        assert_wms_base(
-            "http://localhost/",
-            "http://localhost/wms/df756642-c5a3-4d72-8ad7-629d312ae993",
-        );
-        assert_wms_base(
-            "https://localhost/api/",
-            "https://localhost/api/wms/df756642-c5a3-4d72-8ad7-629d312ae993",
-        );
-        assert_wms_base(
-            "https://www.foobar.de/path/to/wms/",
-            "https://www.foobar.de/path/to/wms/wms/df756642-c5a3-4d72-8ad7-629d312ae993",
```

Comment:
Why did you remove this?
Didn't it check that both root dir as URL and subfolder works?
I remember adding this for a bugfix.

Reply:
it checked that for different base urls the correct wms endpoint URL was created. As the method now takes the base URL directly from the config (like wcs and wfs do, so it's uniform now) we can no longer pass different base URL into the function. I think the sub tests are so trivial that it's ok to remove them. It's not like we implemented the joining of two URLs ourselves but we use the method from the URL crate. So why test what they already tested?

Reply:
7057f65bbb07e00a2bfceaf6c54d0e0e02f3cb3a